hotosm/galaxy-ui
# Add 404 page and improve form context functionality:

**What does this PR do?** 
- This PR adds a 404 page for non-existent routes. As well as pages still under development. 
- It also improves the context functionality used on the Mapathon and UserGroup components. 

**Work done:**
- Combine Mapathon and UserGroup context into one Form context provider.
- Use Form context with Mapathon and User Group components.
- Create a 404 card with a link to existing report pages.
- Create a Default Page component with map background.
- Use the Default Page component for both the Home and 404 pages.
- Redirect to the 404 page for undeclared routes.

**How to test:**
- Git pull and check out branch using `git pull origin && git checkout feature/404-page`
- Install dependencies using `npm i`
- Start the server using `npm start`
- Go to a random route, e.g `/off`

**Screenshot:**
<img width="595" alt="404 " src="https://user-images.githubusercontent.com/31903212/166152231-2d3343e1-47f5-4993-b1db-69a57086bc52.png">

Related issues:
- Fixes #82
